opentech/apply/workflow.py

```diff
@@ -177,6 +201,7 @@ def process(self, phase: 'Phase') -> Union['Phase', None]:
 
 
 class NextPhaseAction(Action):
+    # Change to the next action in the current Stage
```

Comment:
ChangePhase/NextPhaseAction are part of the generic WF, as far as I can see it. Let's clearly demarcate the WF base and the actual OTF WF definitions

Reply:
I moved the visual indicator down a few lines